### ISSUE TITLE
Error on https://unblockedgames.blogbucket.org/electricman-2/

### ISSUE THREAD

**Issue by john249977266** [ruffle-rs/ruffle#6299]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:853:18
    at c.wbg.__wbg_new_df6e6ab7a65c4c4d (https://unblockedgames.blogbucket.org/ruffle/core.ruffle.f09c078c6eb8995999eb.js:1:6964)
    at std::sync::once::Once::call_once::{{closure}}::h4dcb78b97de78bcf (wasm://wasm/0188467a:wasm-function[1020]:0x2f5df3)
    at std::sync::once::Once::call_inner::h9279eef47cf85b81 (wasm://wasm/0188467a:wasm-function[1155]:0x3273c1)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h7e6cf7c840665392 (wasm://wasm/0188467a:wasm-function[4690]:0x4cd85e)
    at std::panicking::rust_panic_with_hook::h47a0e203360b6c10 (wasm://wasm/0188467a:wasm-function[3173]:0x47a972)
    at core::panicking::panic_fmt::h4ca53049f45e3264 (wasm://wasm/0188467a:wasm-function[4957]:0x4d93d3)
    at core::panicking::panic::h35778e3c18266c93 (wasm://wasm/0188467a:wasm-function[4687]:0x4cd385)
    at core::ops::function::FnOnce::call_once::he4326916bb96b1b6 (wasm://wasm/0188467a:wasm-function[1054]:0x30b1a7)
    at ruffle_core::avm1::function::Executable::exec::h3bb94337442bafb9 (wasm://wasm/0188467a:wasm-function[397]:0x1430ab)
    at <ruffle_core::avm1::object::Object as ruffle_core::avm1::object::TObject>::call_method::h74544ea9563647a7 (wasm://wasm/0188467a:wasm-function[466]:0x19f64f)
```

# Player Info
Allows script access: false
Player type: Object
SWF URL: https://files.blogbucket.org/2017/Electric-Man-2-Unblocked.swf
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined

# Page Info
Page URL: https://unblockedgames.blogbucket.org/electricman-2/
SWF URL: https://files.blogbucket.org/2017/Electric-Man-2-Unblocked.swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.82 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-02-13
Channel: nightly
Built: 2022-02-13T00:16:33.556Z
Commit: 9969e0ce5a9313ff4bee98e775afdea78d0adfc0
Is extension: false

**Comment by relrelb** [comment]
Duplicate of #4932.
